_data/simple-icons.json

```diff
@@ -4422,7 +4425,7 @@
         {
             "title": "InVision",
             "hex": "FF3366",
-            "source": "https://www.invisionapp.com/news",
+            "source": "https://in.invisionapp.com/boards/FH3LW3S7XSD/",
             "guidelines": "https://in.invisionapp.com/boards/FH3LW3S7XSD/"
```

Comment:
It always redirects me to https://www.invisionapp.com/resources/brand, so we might use this instead